pyslack/__init__.py

```diff
@@ -41,6 +41,27 @@ def chat_post_message(self, channel, text, **params):
         })
         return self._make_request(method, params)
 
+    def chat_update_message(self, channel, text, timestamp, **params):
+        """chat.update
+
+        This method updates a message.
+
+        Check docs for all available **params options:
+        https://api.slack.com/methods/chat.update
+
+        Required parameters:
+        `ts`:  Timestamp of the message to be updated (e.g: 1405894322.002768)    
```

Comment:
can we add quotes around these `e.g:` params to signify that they're all strings?